core/bentley/src/Id.ts

```diff
@@ -589,6 +589,25 @@ export namespace Id64 {
         for (const lo of entry[1])
           func(lo, entry[0]);
     }
+
+    /** Get an iterator of entries in this set by Id. */
```

Comment:
Consider reverting changes to Id.ts as they will become unused by this PR and I'm not sure we want to encourage people to use Id64Strings with an API designed around pairs of 32-bit integers (it defeats the purpose).

Reply:
reverted all changes to core/bentley